packages/volto/src/components/manage/Blocks/Image/Edit.jsx

```diff
@@ -110,6 +93,7 @@ function Edit(props) {
             block={props.block}
             id={props.block}
             objectBrowserPickerType={'image'}
+            onSelectItem={props.onSelectItem}
```

Comment:
@sneridagh I don't understand this passing of onSelectItem because at least in volto core there is no props.onSelectItem for Image block Edit.jsx. I do understand giving the ability to call this method  if it's passed in the props of ImageWidget.jsx but in the case of Image block it's simply undefined and the fallback is called.

It's nice that you found more things to cleanup such as the extra compose actions!

Reply:
Yes, I agree, I think that it can be removed.

Reply:
@ichim-david @dobri1408 Agreed! yeah, I'll remove it.

Reply:
Done.